src/v/ssx/tests/BUILD

```diff
@@ -62,6 +62,88 @@ redpanda_cc_btest(
     ],
 )
 
+redpanda_cc_btest(
+    name = "sleep_abortable_test",
+    timeout = "short",
+    srcs = [
+        "sleep_abortable_test.cc",
+    ],
+    deps = [
+        "//src/v/base",
+        "//src/v/ssx:sleep_abortable",
+        "//src/v/test_utils:seastar_boost",
+        "@boost//:test",
+        "@seastar",
+        "@seastar//:testing",
+    ],
+)
+
+redpanda_cc_btest(
+    name = "watchdog_test",
+    timeout = "short",
+    srcs = [
+        "watchdog_test.cc",
+    ],
+    deps = [
+        "//src/v/base",
+        "//src/v/ssx:watchdog",
+        "//src/v/test_utils:seastar_boost",
+        "@boost//:test",
+        "@seastar",
+        "@seastar//:testing",
+    ],
+)
+
+redpanda_cc_btest(
+    name = "task_local_ptr_test",
+    timeout = "short",
+    srcs = [
+        "task_local_ptr_test.cc",
+    ],
+    deps = [
+        "//src/v/base",
+        "//src/v/ssx:task_local",
+        "//src/v/test_utils:seastar_boost",
+        "@abseil-cpp//absl/algorithm:container",
+        "@boost//:test",
+        "@seastar",
+        "@seastar//:testing",
+    ],
+)
+
+redpanda_cc_btest(
```

Comment:
CMake uses `cpu = 2` for this test: https://github.com/redpanda-data/redpanda/blob/8d491f3df4a51bd400c1e487012486f98d3fe422/src/v/ssx/tests/CMakeLists.txt#L38

(We default to 4)

Reply:
Using less cpus allows for bazel to parallelize more tests

Reply:
Fixed.

Doesn't that mean that we would be better off with the default set to `1` and increase it only where necessary?